src/pages/ObservationPage.js

```diff
@@ -0,0 +1,31 @@
+import 'react-day-picker/lib/style.css'
+import styled from 'styled-components/macro'
+import ObservationGroup from '../components/ObservationGroup'
+import sortByTime from '../utils/sortByTime'
+
+export default function ObservationPage() {
+  const observations = [
+    { id: 1, time: '8:00', name: 'Blutdruck', value: '94/50' },
+    { id: 2, time: '8:00', name: 'Temperatur', value: '36,5' },
+    { id: 3, time: '8:00', name: 'Gewicht', value: '22,5' },
+    { id: 4, time: '10:00', name: 'FEV1', value: '1,11' },
+  ]
```

Comment:
Are these examples or why are these hard-coded?

Reply:
this user story is just about showing a hard coded list of observations. in the next user story this will be gone and the user can create own observations in a form.